### DISCUSSION TITLE
Test of the OpenSearch search (XML) fails for me

### DISCUSSION THREAD

**Discussion by DJ-SteffEN** [ampache/ampache#4010] [Q&A] [answered]
Hello,

I would like to know if the OpenSearch search (XML) still works, or how I can test it. 

![image](https://github.com/user-attachments/assets/af36878e-7c6e-465e-be94-46e91a09d56a)

Kind regards 
icqtoni

**Comment by lachlan-00** [comment]
For starters it looks like it returns html on my site.
![image](https://github.com/user-attachments/assets/249ca21a-a6b4-4ce9-a82e-a9521c2ebb67)

probably very broken but we should be able to fix it seems like i just need to make it return the correct stuff in xml
https://developer.mozilla.org/en-US/docs/Web/OpenSearch

**Comment by lachlan-00** [comment, marked as answer]
fixed it!
https://github.com/ampache/ampache/issues/4011#issuecomment-2319628590
